validators/btc_address.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+from .utils import validator
+
+pattern = re.compile(r"^[13][a-km-zA-HJ-NP-Z1-9]{25,34}$")
+
+
+@validator
+def btc_address(value):
```

Comment:
Please add a doc block here defining how this function is to be used. See other functions of similar doc blocks. Also reference this doc block in the ``index.rst`` file